onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -460,13 +460,13 @@ void addGlobalMethods(py::module& m, const Environment& env) {
             onnxruntime::CreateExecutionProviderFactory_MIGraphX(0)
 #endif
 #ifdef USE_VITISAI
-            onnxruntime::CreateExecutionProviderFactory_VitisAI("DPU", 0),
+                onnxruntime::CreateExecutionProviderFactory_VitisAI("DPU", 0),
```

Comment:
nit : format, same below

Reply:
This is the formatting I get after running clang-format